code/modules/cargo/packs/emergency.dm

```diff
@@ -206,30 +206,6 @@
 					/obj/item/storage/box/mre/menu4/safe)
 	crate_name = "MRE crate (emergency rations)"
 
-/datum/supply_pack/emergency/syndicate
```

Comment:
This is the one packet change im ok with.

Reply:
Oh dont forget to change OPS GEAR crate or what ever to say something different then `TO CHEAP FOR THAT NULL CRATE HUH?`